src/main.js

```diff
@@ -116,14 +118,36 @@ var currentPoster;
 
 // event listeners go here 👇
 
+showMyPosterButton.addEventListener('click', function(e) {
+  e.preventDefault();
+  designPoster();
+  console.log("Hello!");
+});
+
 showRandomButton.addEventListener('click', updatePoster);
 
 window.addEventListener('load', updatePoster);
+
 makeYourOwnPosterButton.addEventListener("click", showFormSection);
 showSavedPostersButton.addEventListener("click", showSavedPostersSection);
 takeMeBackButton.addEventListener("click", leaveFormSection);
 backToMainButton.addEventListener("click", leaveSavedPostersSection);
 
+// functions and event handlers go here 👇
+
+
+function designPoster() {
+  new Poster(image, title, quote);
+  var image = document.querySelector('#poster-image-url').value;
+  var title = document.querySelector('#poster-title').value;
+  var quote = document.querySelector('#poster-quote').value;
+  images.push(image);
+  titles.push(title);
+  quotes.push(quote);
+  leaveFormSection();
+  showNewPoster(image, title, quote);
+  console.log("hello Stephanie and Mark!")
```

Comment:
Let's delete this console. log. By the way @MTCawthray I believe you can delete these right in github, so they are ready to pull down when we work today.